com.glitterware.passy.yml

```
@@ -0,0 +1,39 @@
+app-id: com.glitterware.passy
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: passy
+separate-locales: false
+finish-args:
+- '--share=ipc'
+- '--socket=fallback-x11'
+- '--socket=wayland'
+- '--device=dri'
+- '--socket=pulseaudio'
+- '--share=network'
+- '--filesystem=home'
+modules:
+- name: passy
+  buildsystem: simple
+  sources:
+  - type: file
+    path: com.glitterware.passy.metainfo.xml
+  - type: file
+    path: passy.svg
+  - type: file
+    path: com.glitterware.passy.desktop
+  - type: archive
+    url: https://github.com/GlitterWare/Passy-Releases/blob/main/versions/v1.2.0/Passy-v1.2.0-Linux-Bundle-No-Updates-Popup.zip?raw=true
+    sha256: 8846eec4d2f19c57be6fc7c2f92513f1a5b843bb26636948a956316e5dfe84eb
+    archive-type: zip
+    dest: bundle
+  build-commands:
+  - mkdir -p /app/Passy
+  - cp -r bundle/* /app/Passy
+  - chmod +x /app/Passy/passy
+  - mkdir -p /app/bin
+  - ln -s /app/Passy/passy /app/bin/passy
+  - install -Dm644 com.glitterware.passy.metainfo.xml -t /app/share/metainfo/
+  - install -Dm644 passy.svg -t /app/share/icons/hicolor/scalable/apps/com.glitterware.passy.svg
```

```suggestion
  - install -Dm644 passy.svg /app/share/icons/hicolor/scalable/apps/com.glitterware.passy.svg
```
I mayb have mistype here. It shouldn't use `-t`